nixos/roles/devhost/default.nix

```diff
@@ -8,13 +8,12 @@ let
   testedRoles = attrNames (lib.filterAttrs (n: v: v.supportsContainers or true) config.flyingcircus.roles);
   excludedRoles = attrNames (lib.filterAttrs (n: v: !(v.supportsContainers or true)) config.flyingcircus.roles);
 
-  enabledContainers = with lib;
-    (filter (container: container.enabled or true)
-           (map
-            (filename: fromJSON (readFile "/etc/devhost/${filename}"))
-            (filter
-              (filename: hasSuffix ".json" filename)
-              (attrNames (readDirMaybe "/etc/devhost/")))))
+  allContainers = with lib;
+     (map
+      (filename: fromJSON (readFile "/etc/devhost/${filename}"))
+      (filter
+        (filename: hasSuffix ".json" filename)
+        (attrNames (readDirMaybe "/etc/devhost/"))))
     # BBB can be removed after a grace period
     ++ (filter (container: container.enabled or true)
```

Comment:
Does it make sense to filter for enabled containers here? The result is called `allContainers`

Reply:
Stupid copy and paste mistake. Refactored.